tests/mock_tables/config_db.json

```diff
@@ -547,7 +547,7 @@
         "NULL": "NULL"
     },
     "INTERFACE|Ethernet0": {
-        "NULL": "NULL"
+        "dhcp_servers@": "12.0.0.1,12.0.0.2,12.0.0.3"
```

Comment:
This is not according to the existing schema. INTERFACE tables does not have `dhcp_servers` as attribute . Are you proposing a schema change? Currently for vlans, it is part of VLAN table and not VLAN_INTERFACE

Reply:
Yes, "dhcp_servers" is an L3 attribute, so added it to INTERFACE and PORTCHANNEL_INTERFACE tables.
For Vlan case, still using VLAN table only.

Reply:
It will be inconsistent with VLAN_INTERFACE. Fine with me if @tahmed-dev, @jleveque and @lguohan sign off.

Reply:
I agree with @prsunny. Lumping dhcp_servers with the interface is inconsistent with the current. I think since [PR:6221](https://github.com/Azure/sonic-buildimage/pull/6221) attempts to have DHCP relay preside on almost ever interface, it would be better to have separate table for DHCP_RELAY configuration. @lguohan, @jleveque what are your thoughts on that?

Reply:
I think this is debatable. Previously we considered the list of DHCP server IPs to be an attribute of a VLAN table entry. This approach is consistent with the current approach, extending it to INTERFACE and PORTCHANNEL_INTERFACE tables. We could take your approach, but this would require refactoring to create a DHCP_RELAY table and I guess have the attributes be the interface names? Basically inverting this approach?

Reply:
It is indeed. I am thinking of having just one binary to serve as DHCP relay down the road and so having those entries consolidated under DHCP_RELAY table would be self contained and eliminate the need to search multiple tables. 

Yes, the attributes would be interface name serving as a key and a downlink at the same time and the map/table would contain uplink interfaces and DHCP helper IPs (helper IPs could be per DHCP feature.)

Reply:
Is there a conclusion here please guys?

Reply:
If we add new table like "DHCP_RELAY" table, then we need to change the supervisord.conf file also accordingly, it might be big change for now.

Can we move the dhcp_servers from VLAN table to VLAN_INTERFACE table to be more inconsistent ??